src/components/chatBar/ChatBar.css

```
@@ -0,0 +1,308 @@
+    height: 450px;
+    background-color: #242424;
+    border: 1px solid #333;
+    border-radius: 12px 12px 0 0;
+    display: flex;
+    flex-direction: column;
+    box-shadow: 0 -5px 25px rgba(0, 0, 0, 0.4);
+    overflow: hidden;
+    animation: slideUp 0.3s ease-out;
+}
+
+@keyframes slideUp {
```

Stylelint (17.3.0)

[error] Expected keyframe name "slideUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)